uktrade/data-hub-frontend
# Add new design for interaction activities

## Description of change

As part of the ongoing work to re-design the contact activity feed, the interaction card has been redesigned. This change is only visible for users with the relevant feature flag enabled and does not affect the company feed (or any other ActivityFeeds).

<img width="736" alt="Screenshot 2022-04-22 at 15 20 02" src="https://user-images.githubusercontent.com/36161814/164733336-d0d407ab-fe67-4148-9ad6-9e958815636a.png">

## Test instructions

With the `user-contact-activities` flag enabled and your local FE connected to the dev environment, go to http://localhost:3000/contacts/715e8a58-6515-4d52-a4b5-84cb49566b01/interactions?featureTesting=user-contact-activities. You should see various interactions showcasing various options available with the new design.

## Screenshots
If the feature flag is disabled, no changes will be visible.

### Before (with feature flag enabled)

<img width="1001" alt="Screenshot 2022-04-22 at 15 13 02" src="https://user-images.githubusercontent.com/36161814/164733610-625ad1f1-d3ea-47cb-927b-aec96f856f1e.png">


### After

<img width="990" alt="Screenshot 2022-04-22 at 15 20 15" src="https://user-images.githubusercontent.com/36161814/164733668-6979b1b2-fcb9-4cb9-a83d-880b63a226ac.png">


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
